kafka-connect/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:25
+
+RUN dnf -y install which java-1.8.0-openjdk libaio python gettext hostname iputils wget && dnf clean all -y
```

Comment:
The rest I can understand, but out of interest why do we need `which`, `libaio`, `python` and `gettext`?

Reply:
My guess ... 
* gettext container contains envsubst used in some of the scripts
* hostname is also used in some of the scripts
* I actually don't think Kafka is using libaio
* Nothing seems to use which, python or iputils.

@ppatierno Are these just copy paste errors from some other EnMasse images (Artemis is using libaio)? Or did you include these intentionally?

Reply:
@scholzj you are right for the first two. To be honest I don't remember about the others and maybe a copy/paste.

Reply:
Ok, I suggest to add this to the #13 issue. Wdyt @tombentley and @ppatierno?